### ISSUE TITLE
Update fontawesome to 5.1

### ISSUE THREAD

**Issue by ertrzyiks** [hexojs/hexo-fontawesome#23]
```
npm WARN deprecated @fortawesome/fontawesome-free-solid@5.0.13: This package is deprecated. See https://git.io/fNCzJ for information about upgrading.
npm WARN deprecated @fortawesome/fontawesome-free-brands@5.0.13: This package is deprecated. See https://git.io/fNCzJ for information about upgrading.
npm WARN deprecated @fortawesome/fontawesome-free-regular@5.0.13: This package is deprecated. See https://git.io/fNCzJ for information about upgrading.
npm WARN deprecated @fortawesome/fontawesome@1.1.8: This package is deprecated. See https://git.io/fNCzJ for information about upgrading.
```
https://github.com/FortAwesome/Font-Awesome/blob/master/UPGRADING.md#50x-to-510

**Comment by weyusi** [comment]
fontawesome@5 introduces breaking change ([examples](https://github.com/geekplux/hexo-theme-typing/pull/28#issuecomment-428412864) here). `regular` icons are no longer free.
Alternative is to use [fork-awesome](https://github.com/ForkAwesome/Fork-Awesome) which retains compatibility with fontawesome@4.

**Comment by itpropro** [comment]
Any news to this? The update by fontawesome is from April 2018, can anyone confirm that this plugin is actually still in development?

**Comment by ertrzyiks** [comment]
@itpropro yup this is plugin is still alive. I noticed the problem only two weeks ago and need to find time to address it, no worries.

**Comment by ertrzyiks** [comment]
The migration happens in #25 . I expect it to be published next week.

**Comment by ertrzyiks** [comment]
Released as 2.0.0
